components/centos/centos-openshift-setup/Vagrantfile

```diff
@@ -5,29 +5,18 @@
 DOCKER_REGISTRY="docker.io"
 
 # The name of the OpenShift image.
-ORIGIN_IMAGE_NAME="openshift/origin:v1.1.1"
+ORIGIN_IMAGE_NAME="openshift/origin"
```

Comment:
Do we need to allow this value here if we hard code origin for the sub-components below?

Alternately, lines 56-59 should use this tag.